src/core/command_replacements.go

```diff
@@ -59,6 +59,8 @@ var locationsReplacement = regexp.MustCompile(`\$\(locations ([^\)]+)\)`)
 var exeReplacement = regexp.MustCompile(`\$\(exe ([^\)]+)\)`)
 var outExeReplacement = regexp.MustCompile(`\$\(out_exe ([^\)]+)\)`)
 var outReplacement = regexp.MustCompile(`\$\(out_location ([^\)]+)\)`)
+var absOutExeReplacement = regexp.MustCompile(`\$\(abs_out_exe ([^\)]+)\)`)
```

Comment:
Hmmmm, I'm not sure I like these becoming a replacement any build rule has access to. It seems like if it were used in an actual build rule it would only be doing bad things.
Can we just use `filepath.Abs` on the returned path in run_step.go?